10_Examples/will_onboarding1/include/will_onboarding1/Mather.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <chrono>
+#include <memory>
+#include <string>
+
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/int32.hpp"
+
+using namespace std::chrono_literals;
+using namespace std;
+using std::placeholders::_1;
+
+namespace Math_Space {
```

Comment:
we do snake case for namespaces, i.e. math_space.

Also putting space at the end of namespace is a bit odd.

Reply:
the space after the namespace comes from the auto formatter.